packages/shared/src/components/ConditionalWrapper.tsx

```diff
@@ -13,3 +13,14 @@ const ConditionalWrapper = ({
   condition ? wrapper(children) : (children as ReactElement);
 
 export default ConditionalWrapper;
+
+export const ConditionalRender = ({
+  condition,
+  children,
+}: Pick<ConditionalWrapperProps, 'condition' | 'children'>): ReactNode => {
+  if (!condition) {
+    return null;
+  }
+
+  return children;
+};
```

Comment:
I wanted to avoid the ugly ternary ladder, which adds indent and confusing conditions.

For example:

```
{!preselected &&
        (selected ? (
          <Component1 />
        ) : (
          <Component2 />
        )
      )
}
```

Reply:
It doesn't really though right, it's just masking it.
Technically still read it as if...else inside if...else

Not against it, but not sure it's that much clearer.
@capJavert // @omBratteng what you guys think?

I'm happy to keep it though to unblock you.

Reply:
Probably more of a cosmetic than functional change for readability.

Reply:
Ah agreed I'd love to avoid the nested ternaries in the first place.
But to me this is just a wrapper the result is still having the ternary.

One other solution is to render 2 functions and do the inside ternary inside 1 function?

Reply:
Yeah, that can work too, by doing early returns. I'll be merging this now but I'll update and raise another PR once we get more answers on what could fit our current setup.